sbat.c

```diff
@@ -335,4 +335,62 @@ parse_sbat_var(list_t *entries)
 	FreePool(data);
 	return EFI_INVALID_PARAMETER;
 }
+
+EFI_STATUS
+init_sbat()
+{
+	EFI_STATUS efi_status = EFI_SUCCESS;
+	UINT32 attributes = 0;
+
+	CHAR8 *sbat;
+	UINTN sbatsize = 4096;
```

Comment:
what happens if the SBAT variable size is bigger than 4 KiB ?

Reply:
Technically we don't care about anything more than strlen(SBAT_INIT) +1, but I'm just testing the size and allocating dynamically now.